src/diffusers/image_processor.py

```diff
@@ -95,8 +95,11 @@ class VaeImageProcessor(ConfigMixin):
             `height` and `width` arguments from [`image_processor.VaeImageProcessor.preprocess`] method.
         vae_scale_factor (`int`, *optional*, defaults to `8`):
             VAE scale factor. If `do_resize` is `True`, the image is automatically resized to multiples of this factor.
-        resample (`str`, *optional*, defaults to `lanczos`):
-            Resampling filter to use when resizing the image.
+        resample (`str`, *optional*, defaults to `"lanczos"`):
```

Comment:
can we add a new `WanVaeImageProcessor(VaeImageProcessor)` and put into wan folder, under `utils.py` file I think? 
(we start to see more and more custom preprocess methods, almost everyone does and they don't really get reused across models, I think moving forward let's just do this for all new models)

cc @DN6 here too, let me know what you think